doc/_resource_types/alb.md

```diff
@@ -37,3 +37,11 @@ describe alb('my-alb') do
   it { should belong_to_vpc('my-vpc') }
 end
 ```
+
+### have_tag
```

Comment:
Could you remove this segment ? ( 
Duplicate in https://github.com/k1LoW/awspec/pull/472/files#diff-f78ddd4a16af4e4b3cb621d9d3360879R147 and https://github.com/k1LoW/awspec/pull/472/files#diff-f78ddd4a16af4e4b3cb621d9d3360879R155 )